src/traceradvectiondiffusion.jl

```diff
@@ -47,6 +50,37 @@ function Problem(dev;
   return FourierFlows.Problem(equation, stepper, dt, grid, vars, params, dev)
 end
 
+"""
+    Problem(MQGprob; parameters...)
+
+Construct a constant diffusivity problem with a turbulent flow from the `GeophysicalFlows.jl` package.
+"""
+function Problem(dev, MQGprob::FourierFlows.Problem;
+                κ = 0.1,
+                η = κ,
+          stepper = "FilteredRK4",
+   tracer_release = 0.0
+  )
+  
+  nlayers = typeof(MQGprob.params) <: SingleLayerParams ? 1 : 
+            typeof(MQGprob.params) <: TwoLayerParams ? 2 : MQGprob.params.nlayers
+
+  grid = MQGprob.grid
+  if tracer_release != 0
+    @info "Stepping the flow forward"
+    while MQGprob.clock.t < tracer_release
+        MultiLayerQG.stepforward!(MQGprob)
+        MultiLayerQG.updatevars!(MQGprob)
+    end
+  end
```

Comment:
Would this simplify things here?

https://github.com/FourierFlows/FourierFlows.jl/blob/75402b443adbf8b44cd3f32eb3bf80644496509c/src/timesteppers.jl#L606

Reply:
Yep that would be great! I will make the change now.

Reply:
note that it won't work with ETDRK4...

also, it may result in changing the dt

E.g., if dt=0.1 and we ask `step_until(prob, 1.034)` then it'll timestep up to t=1 and then change `dt=0.034`....

Reply:
oh no, my bad regarding `dt`... in the end, `step_until!` changes back the dt to the original one